mono/metadata/sgen-bridge.c

```diff
@@ -655,19 +668,24 @@ register_test_bridge_callbacks (const char *bridge_class_name)
 gboolean
 sgen_bridge_handle_gc_debug (const char *opt)
 {
+	assert (!bridge_processor_started ());
+
 	if (g_str_has_prefix (opt, "bridge=")) {
 		opt = strchr (opt, '=') + 1;
 		register_test_bridge_callbacks (g_strdup (opt));
 	} else if (!strcmp (opt, "enable-bridge-accounting")) {
-		bridge_processor.enable_accounting ();
+		bridge_processor_config.accounting = TRUE;
+	} else if (!strcmp (opt, "enable-tarjan-precise-merge")) {
```

Comment:
I'm filing a request with the docs team to get it documented in the appropriate place for Android.